sport/app/football/conf/context.scala

```diff
@@ -46,19 +59,24 @@ class FootballLifecycle(
     Ideally, we should combine the logic of these two jobs so we only call once. This is tricky to achieve with the current code.
     We intend to move to a stream model so we can achieve this.
      */
-    jobs.schedule("MatchDayAgentRefreshJob", "0 0/5 * * * ?") {
+    // if preview "Every hour at 40 minutes" otherwise "Every 5 minutes"
+    // 40 minute was chosen for preview to run these jobs after the 35 CompetitionAgentRefreshJobs are finished
+    jobs.schedule("MatchDayAgentRefreshJob", if (context.isPreview) "0 40 * * * ?" else "0 0/5 * * * ?") {
       competitionsService.refreshMatchDay(defaultClock)
     }
 
-    jobs.schedule("MaybeMatchDayAgentRefreshJob", "0 0/1 * * * ?") {
+    // if preview "Every hour at 40 minutes" otherwise "Every minute"
+    jobs.schedule("MaybeMatchDayAgentRefreshJob", if (context.isPreview) "0 40 * * * ?" else "0 0/1 * * * ?") {
       competitionsService.maybeRefreshLiveMatches(defaultClock)
     }
 
-    jobs.schedule("CompetitionRefreshJob", "0 0/10 * * * ?") {
+    // if preview "Every hour at 40 minutes" otherwise "Every 10 minutes"
+    jobs.schedule("CompetitionRefreshJob", if (context.isPreview) "0 40 * * * ?" else "0 0/10 * * * ?") {
       competitionsService.refreshCompetitionData()
     }
 
-    jobs.schedule("TeamMapRefreshJob", "0 0/10 * * * ?") {
+    // if preview "Every hour at 40 minutes" otherwise "Every minute"
```

Comment:
Should this comment say every 10 minutes?

```suggestion
    // if preview "Every hour at 40 minutes" otherwise "Every 10 minutes"
```